### PR TITLE
LPS-186301 Add Editable Action Panel

### PR DESCRIPTION
**Pull request by alessandromalves**
https://issues.liferay.com/browse/LPS-186301
cc.: @sandrodw3 @david-gutierrez-mesa

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by alessandromalves** [comment]
ci:test:sf

**Comment by alessandromalves** [comment]
ci:test:relevant

**Comment by alessandromalves** [comment]
This on hold since this can't be merged before https://github.com/liferay-echo/liferay-portal/pull/12332

**Comment by alessandromalves** [comment]
Resent based on https://github.com/liferay-echo/liferay-portal/pull/12317

**Comment by Tim-Cao** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/fragments/Fragment.path [hunk 1]
```diff
@@ -284,7 +284,7 @@
 </tr>
 <tr>
 	<td>FRAGMENT_CONFIGURATION_TYPE</td>
-	<td>//div[contains(@class,'component-${key_fragmentName}')]//*[self::a or self::button][contains(@class,'-${key_type}')]</td>
+	<td>//div[contains(@class,'component-${key_fragmentName}')]//*[self::a or self::button][contains(@class,'-${key_type}')] | //div[contains(@class,'component-${key_fragmentName}')]//*[self::a or self::button][contains(@data-lfr-editable-type,'${key_type}')]</td>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/fragments/Fragment.path:287]: This change is not correct. The button type should be defined in class btn-link

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/12358

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**